crypto/x509/x509_vfy.c

```diff
@@ -1863,10 +1863,14 @@ int X509_cmp_time(const ASN1_TIME *ctm, time_t *cmp_time)
             return 1;
     }
     i = strcmp(buff1, buff2);
+    return i;
+
+#if 0
     if (i == 0)                 /* wait a second then return younger :-) */
-        return -1;
+        return 0;
     else
         return i;
+#endif 
```

Comment:
You're changing semantics of a public function.  That throws this change to 1.2.0.  Is that what you desire?